src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Lottery shouldn't extend ColorSupplier. You should create an object of ColorSupplier and use it
```
ColorSupplier colorSupplier = new ColorSupplier();
...
colorSupplier.getRandomColor();
```